proto/v3/message_contents/public_key.proto

```diff
@@ -4,8 +4,9 @@ syntax = "proto3";
 
 package xmtp.v3.message_contents;
 
-// v4 here since it was already v3 for golang prior to this
-option go_package = "github.com/xmtp/proto/v4/go/message_contents";
+import "v3/message_contents/association.proto";
+
+option go_package = "github.com/xmtp/proto/v3/go/v3/message_contents";
```

Comment:
The `go_package` field is sensitive. It can't be `github.com/xmtp/proto/v4` unless we increment the version on the git tag to a new major version. Until then, needs to stay as `v3`.